src/scripts/resources/airlock-lib.sh

```diff
@@ -65,13 +65,18 @@ function cleanUpHistory() {
     abortOnError
 
     echo "activate configuration..."
-    ${CURL} "https://${AIRLOCK}/airlock/rest/configuration/configurations/activate" \
+    STATUS_CODE=`${CURL} "https://${AIRLOCK}/airlock/rest/configuration/configurations/activate" \
+        -w "%{http_code}" \
```

Comment:
this does not work generally, since the (potential) body is also printed to stdout. you must use
`curl -s -o /dev/null -w "%{http_code}" ...`
you could use the curl "--fail" flag an check the return code of curl. This is not 'fail-safe' as the man page states, but most likely  'good enough' here.